pom.xml

```diff
@@ -362,6 +362,7 @@
                 <artifactId>maven-javadoc-plugin</artifactId>
                 <version>${maven-javadoc-plugin.version}</version>
                 <configuration>
+					<sourcepath>src/main/java</sourcepath>
```

Comment:
Could you indent this to be 'inside' configuration?

Reply:
Oh, yes, I guess my IDE is configured differently